packages/storybook/stories/ChatComposite/snippets/Container.snippet.tsx

```diff
@@ -3,10 +3,31 @@ import {
   CommunicationUserIdentifier,
   getIdentifierKind
 } from '@azure/communication-common';
-import { ChatAdapter, ChatComposite, createAzureCommunicationChatAdapter } from '@azure/communication-react';
+import {
+  ChatAdapter,
+  ChatComposite,
+  createAzureCommunicationChatAdapter,
+  AvatarPersonaDataProvider,
+  AvatarPersonaData
+} from '@azure/communication-react';
 import { Theme, PartialTheme } from '@fluentui/react';
+import _ from 'lodash';
 import React, { useState, useEffect, useMemo } from 'react';
 
+// Sample usage of Custom Data Provider for AvatarPersona with Memoization.
```

Comment:
This piece of code is here as an example code for PR reviewers.
I will remove this once I move this PR out of a draft.

Question for folks. How should we best showcase this to CONTOSO that they need to take care of memoization themselves (possibly using this mechanism)?